include/glow/Importer/ONNX.h

```diff
@@ -48,28 +48,47 @@ class ONNXModelLoader
   /// Load the network initializers from the GraphProto.
   void loadInitializers(onnx::GraphProto &net);
 
+  /// \returns true if operator \p op can be loaded.
   /// Load the operator \p op into the network. This creates one or more nodes
   /// in the network.
-  void loadOperator(const onnx::NodeProto &op);
+  bool loadOperator(const onnx::NodeProto &op);
 
-  /// Reads a network (weights or structure) from the serialized protocol buffer
-  /// file.
-  bool loadProtoFile(onnx::GraphProto &net, const std::string &filename);
+  /// \returns true if serialized protobuf can be read from the \p filename.
+  /// Loads GraphProto \p net containing network (weights or structure)
+  /// from the serialized protobuf file.
+  bool loadProto(onnx::GraphProto &net, const std::string &filename);
+
+  /// \returns true if serialized protobuf can be constructed from the in-memory
+  /// serialized protobuf.
```

Comment:
The first sentence sounds strange, because we construct a serialized protobuf from a serialized protobuf. That should always work.
I am guessing we meant to say that we manage to build \p net from the in-memory protobuf.

Reply:
indeed!